src/emulator/modules/SceNet/src/SceNet.cpp

```diff
@@ -241,18 +244,19 @@ EXPORT(Ptr<const char>, sceNetInetNtop, int af, const void *src, Ptr<char> dst,
 #else
     const char *res = inet_ntop(af, src, dst_ptr, size);
 #endif
-    if (res == nullptr){
```

Comment:
I think you still want to check this, and return the equivalent null Ptr:

`Ptr<char>()`